react/features/base/tracks/actions.js

```diff
@@ -1,3 +1,4 @@
+import VideoLayout from '../../../../modules/UI/videolayout/VideoLayout';
```

Comment:
I haven't tested it, but this may break mobile.

Reply:
Wow very good catch! I'm actually even using parts of VideoLayout in an action that runs on mobile....

Will fix it!